apps/fyler/src/fyler_monitor.erl

```diff
@@ -43,7 +43,8 @@ stop_monitor() ->
 init(_Args) ->
   ?D(monitor_init),
   Pid = spawn_link(fyler_system_listener, listen, []),
-  {ok, idle, #state{listener = Pid, cpu_max = ?Config(cpu_high_watermark, 90), timeout = ?Config(cpu_check_timeout, 10000)}}.
+  State = #state{listener = Pid, cpu_max = ?Config(cpu_high_watermark, 90), timeout = ?Config(cpu_check_timeout, 5000)},
```

Comment:
Line 46 is too long:   State = #state{listener = Pid, cpu_max = ?Config(cpu_high_watermark, 90), timeout = ?Config(cpu_check_timeout, 5000)},.